client/src/setupProxy.js

```diff
@@ -1,11 +1,11 @@
 const { createProxyMiddleware } = require('http-proxy-middleware');
 
 module.exports = function(app) {
-  app.use(
-    '/api',
-    createProxyMiddleware({
-      target: 'http://localhost:3001',
-      changeOrigin: true,
-    })
-  );
+    app.use(
+        '/api',
+        createProxyMiddleware({
+            target: 'http://server:3001',
```

Comment:
I think you are going to need to variablize this otherwise this will not work locally without docker.  Ideally it should work either way depending on config.

Reply:
I agree - we can probably do this once we move it to the CMS repo though